R/subset.R

```diff
@@ -58,14 +61,36 @@ subset.ddpcr_plate <- function(x, wells, samples, ...) {
       plate_meta(x) %>%
       dplyr::filter_(~ sample %in% samples) %>%
       .[['well']]
+  } else if (!missing(targets_ch1) | !missing(targets_ch2)) {
+    wells <-
+      plate_meta(x) %>%
+      .[['well']]
   } else {
     return(x)  # if no arguments, just return the same plate
   }
   
+  if (!missing(targets_ch1)) {
+    wells_from_targets <-
+      plate_meta(x) %>%
+      dplyr::filter_(~ target_ch1 %in% targets_ch1) %>%
```

Comment:
Does this code work ok without error if the user gives targets that are not actually in the data?